exporters/prometheus/exporter.go

```diff
@@ -227,7 +231,10 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 			if typ == nil {
 				continue
 			}
-			name := c.getName(m)
+			name, err := c.getName(m)
+			if err != nil {
+				continue
```

Comment:
can we add a TODO linking to https://github.com/open-telemetry/opentelemetry-go/issues/7066 to handle this error gracefully?

Reply:
and then just continue? I am unfamiliar with otel.Handle so I'm not sure of the use pattern.  (It looks like that's the thing to do)